mzLib/Test/DataFiles/centroid_1x_MS1_4x_autoMS2.d/demo_line_for_austin.m/submethods.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<method>
+<submethods>


```

Comment:
intended newline characters?

Reply:
All these are in the test Bruker files, directly from the instrument, with no modifications. I don't want to change them, and tests are passing. Not sure what Bruker is doing.